docs/en_US/Tutorial/QuickStart.rst

```diff
@@ -4,7 +4,7 @@ QuickStart
 Installation
 ------------
 
-We currently support Linux, macOS, and Windows. Ubuntu 16.04 or higher, macOS 10.14.1, and Windows 10.1809 are tested and supported. Simply run the following ``pip install`` in an environment that has ``python >= 3.6``.
+Currently, NNI supports running on Linux, MacOS and Windows. Ubuntu 16.04 or higher, macOS 10.14.1, and Windows 10.1809 are tested and supported. Simply run the following ``pip install`` in an environment that has ``python >= 3.6``.
```

Comment:
It's `macOS`.

Reply:
Thanks for pointing it out, I will update it.